### PR TITLE
V100 gem warnings

### PR DESCRIPTION
**Pull request by ssayer**
Fix some warnings when building the gem. Locks down dependencies somewhat.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [ ] JIRA ticket for this PR:
- [ ] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [ ] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced. (N/A)
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [ ] Code coverage has not gone down and all code touched or added is covered.
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here:
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here:
- [ ] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [ ] JIRA test links:
- [ ] Justification for using JIRA tests:
- [ ] JIRA tests have been added to sprint

**Reviewer 1:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

### CODE REVIEW

File: cql_qdm_patientapi.gemspec [hunk 1]
```diff
@@ -9,16 +9,17 @@ Gem::Specification.new do |spec|
   spec.authors     = ['The MITRE Corporation']
   spec.email       = ['tacoma-list@lists.mitre.org']
 
-  spec.summary     = 'Interface from HDS patient model to QDM CQL execution engine'
+  spec.summary     = 'QDM CQL execution engine'
```
Comment (mayerm94) [inline cql_qdm_patientapi.gemspec]: app/assets/javascripts/datatypes still contains the model interface between the execution engine and HDS patient model, are these models going to pulled out into their own repository in the future? As is, I would think the summary should read "QDM CQL execution engine and HDS patient model interface"

File: cql_qdm_patientapi.gemspec [hunk 2]
```diff
@@ -9,16 +9,17 @@ Gem::Specification.new do |spec|
...
   spec.description = 'Interface from HDS patient model to QDM CQL execution engine'
```
Comment (losborne) [inline cql_qdm_patientapi.gemspec]: Should this line be updated to match `spec.summary`?
